servlet/build.xml

```diff
@@ -43,13 +42,28 @@
         <exclude name="com/google/gwt/json/**" />
         <exclude name="com/google/gwt/junit/*" />
         <exclude name="com/google/gwt/junit/client/GWTTestCase.*" />
-        <exclude name="com/google/gwt/junit/remote/**" />
         <exclude name="com/google/gwt/junit/server/**" />
-        <exclude name="com/google/gwt/benchmarks/*" />
-        <exclude name="**/*.gwtar" />
       </fileset>
-
-      <!-- additional dependencies (used by scripts). -->
```

Comment:
This was a placeholder added in https://github.com/gwtproject/gwt/commit/498861e7f00d4b89f23d7bc0900f295b46a160e9, but was never used (at least in open source).